### ISSUE TITLE
Snippet does not show up after the key combination

### ISSUE THREAD

**Issue by Demers94** [jeremyvii/vs-docblockr#13]
**Describe the bug**

When I type the `/**` characters above the first line of a method, the snippet completion window does not show up. Instead, VSCode automatically add the closing `*/` to the end of the line.

Is this due to a setting in VSCode?

**To Reproduce**
1. Put your cursor above the opening line of a method
2. Type `/**` 
3. Notice that the snippet window does not appear and pressing `TAB` does not expand the docblock.

**Code Snippet**
```
public static function indexQuery(NovaRequest $request, $query)
    {
        return $query->where('is_company', true);
    }
```

The problem is however not limited to this snippet, I tried with many different methods in different files, it does not work with JS files either.

**Screenshots**
![](https://media.giphy.com/media/B31CYwCvy1B8zqIzVb/giphy.gif)

**Enviroment (please complete the following information):**
 - OS: Windows 10 1803 
 - Version 0.4.2 of the extension
 - Language(s) : PHP and Javascript

**Additional context**

Here is my settings file in case anything here is interfering with the extension : 
https://pastebin.com/JDJZqYGc

**Comment by jeremyvii** [comment]
For the snippet you sent above, it appears the PHP parser doesn't account for class types. I should be able to work this out in a day or so. But regarding the JavaScript issue, could you send me a few examples of code snippets that are failing? Just so I know what to look for and can write the appropriate test cases to account for them.

**Comment by Demers94** [comment]
After reloading VSCode a few times I got the plugin to work with all the other methods (both PHP & JS) that I was having trouble with yesterday, it seems that there was an issue with VSCode not reloading properly or something like that.

It still didn't work for the above snippet but you already explained why. Thank you for looking into it!

**Comment by jeremyvii** [comment]
This should be fixed once updated to to release [0.4.3](https://github.com/jeremyvii/vs-docblockr/releases/tag/0.4.3).
